Alignment/OfflineValidation/python/TkAlAllInOneTool/GCP_Ntuples_cfg.py

```diff
@@ -0,0 +1,81 @@
+import FWCore.ParameterSet.Config as cms
+#import FWCore.PythonUtilities.LumiList as LumiList
+
+from FWCore.ParameterSet.VarParsing import VarParsing
+
+from Alignment.OfflineValidation.TkAlAllInOneTool.utils import _byteify
+
+import json
+import os
+
+##Define process
+process = cms.Process("ValidationIntoNTuples")
+
+##Argument parsing
+options = VarParsing()
+options.register("config", "", VarParsing.multiplicity.singleton, VarParsing.varType.string , "AllInOne config")
+
+options.parseArguments()
+
+###Set validation mode
+#valiMode = "StandAlone"
+
+##Read in AllInOne config in JSON format
+with open(options.config, "r") as configFile:
+    config = _byteify(json.load(configFile, object_hook=_byteify),ignore_dicts=True)
```

Comment:
I thought the `_byteify` was not needed anymore?

Reply:
That was removed to make runtests work.